src/actions/publish.js

```diff
@@ -95,6 +95,11 @@ async function storeJustPublishedProfileData(
   sanitizedInformation,
   prepublishedState: State
 ): Promise<void> {
+  if (process.env.NODE_ENV === 'test' && !window.indexedDB) {
+    // bailout early in tests
+    return;
+  }
```

Comment:
Maybe that's not necessary now. I added this before I added the fake indexeddb for the symbolication itself. But this also does no harm. What do you think?

Reply:
Also this new branch is the reason `publish.js` coverage decreases => actually no test is using it.